appveyor.yml

```diff
@@ -26,6 +26,15 @@
     - cmd: >-
         subst s: c:\projects
     - ps: set-location -path s:\node_modules\node-sass
+    - ps: Uninstall-Product node 4 $env:platform
+    - ps: Write-Host "Installing node-chakracore..." -ForegroundColor Cyan
+    - ps: Write-Host "Downloading..."
+    - ps: $msiPath = "$env:USERPROFILE\node-chakracore-$env:platform.msi"
+    - ps: $file = "https://github.com/nodejs/node-chakracore/releases/download/node-chakracore-7.0.0-pre9/node-chakracore-$($env:platform).msi"
```

Comment:
@xzyfer , what is the expectation here when we release new binary? Will we have to update this version? How does it work for `nodev8` when new versions are released?